build/build.proj

```diff
@@ -337,6 +364,27 @@
 
   <!--
     ============================================================
+    Restore Apex 
+    ============================================================
+  -->
+  <Target Name="RestoreApex">
```

Comment:
does the ApexTestsStandalone target need to depend on this target or both need to be run separately?